apps/webapp/app/header/tablet-sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+import {
+  NavigationMenu,
+  NavigationMenuContent,
+  NavigationMenuItem,
+  NavigationMenuLink,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+  navigationMenuTriggerStyle,
+} from '@penumbra-zone/ui';
+import Link from 'next/link';
+import { headerLinks } from './constants';
+import { cn } from '@penumbra-zone/ui/lib/utils';
+import { useTypedPathname } from '../../hooks/typed-pathname';
+import { DappPath } from '../../shared/header/types';
+
+export const TabletSidebar = () => {
```

Comment:
When it's first open, it's empty for a half second. Please fix.

<img width="226" alt="Screenshot 2023-11-07 at 3 05 49 PM" src="https://github.com/penumbra-zone/web/assets/16624263/32001b44-3482-4af6-a3a5-ee69a7b43f04">